config/application.rb

```diff
@@ -44,5 +44,8 @@ class Application < Rails::Application
 
     ManageIQ::API::Common::Logging.activate(config)
     ManageIQ::API::Common::Metrics.activate(config, "topological_inventory_api")
+
+    require Rails.root.join("app/middleware/tracker_middleware").to_s
```

Comment:
Why app/middleware, as opposed to lib/middleware?

Reply:
Also we shouldn't be using require on things in autoload dirs.  At a minimum we should use `require_dependency`, or just let autoload load it (if possible)